lingon/src/renderer.rs

```diff
@@ -248,15 +224,20 @@ pub type RenderFn = dyn FnMut(
     &mut GL33Surface,
 ) -> Result<(), ()>;
 
+/// A way to handle random variables.
 pub enum Distribution {
+    /// All values are equally likely, no bias.
     Uniform,
+    /// The sum of 2 uniform samples, creates a bias towards the middle.
     TwoDice,
+    /// The sum of 3 uniform samples, creates a stronger bias towards the middle.
     ThreeDice,
```

Comment:
```suggestion
    /// All values are equally likely with no bias.
    Uniform,
    /// The sum of 2 uniform samples with a bias towards their average.
    TwoDice,
    /// The sum of 3 uniform samples with a bias towards the second value.
    ThreeDice,
```

Reply:
![image](https://user-images.githubusercontent.com/19999497/113035144-34622780-9193-11eb-9378-70289a0d391f.png)
Joel does not agree.